docker/jupyter_notebook_config.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+
+from IPython.lib import passwd
+
+c.NotebookApp.ip = '*'
```

Comment:
Is there a convenient way of allowing the user to run Jupyter remotely?  Would ideally like to spin up clusters in the cloud and connect them to Grid with this container, and I think that would require being able to connect to Jupyter through ssh or through some secure static hostname/IP. Would like to pass a few flags to enable this, default should definitely have the Jupyter running from localhost.

Reply:
Tell me if I understand incorrectly, but i guess you can already do that. Let say you spin up the worker using **run_grid** in the cloud. Then you can access the notebook in you browser using IP_OF_YOUR_INSTANCE:8888